apps/data-pipeline/ap-exams-importer/src/names.ts

```diff
@@ -0,0 +1,21 @@
+// https://apstudents.collegeboard.org/courses
+
+export default {
+  "AP CALCULUS AB": "AP Calculus AB",
+  "AP CALCULUS AB SUB": "AP Calculus BC, Calculus AB subscore",
+  "AP CALCULUS BC": "AP Calculus BC",
+  "AP CHEMISTRY": "AP Chemistry",
+  "AP COMP SCI A": "AP Computer Science A",
+  // this hasn't been offered since 2009
+  "AP COMP SCI AB": "AP Computer Science AB",
```

Comment:
Given that this hasn't been offered in 2009 and it looks similar to CSA, we should consider adding an indication to the human-readable name. Perhaps something like "AP Computer Science AB (last offered 2009)".

Reply:
But then this wouldn't be the official name from College Board and we are now injecting opinion.